tests/test_vllm.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
+# -- end do not touch
+
+import unittest  # noqa: E402
+from vllm import LLM, SamplingParams  # noqa: E402
+from gptqmodel import BACKEND, GPTQModel  # noqa: E402
+
+class TestLoadVLLM(unittest.TestCase):
+    MODEL_ID = "LnL-AI/TinyLlama-1.1B-Chat-v1.0-GPTQ-4bit"
+    prompts = [
+        "Hello, my name is",
+        "The president of the United States is",
+        "The capital of France is",
+        "The future of AI is",
+    ]
+    sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
+    def test_load_vllm(self):
+        model = GPTQModel.from_quantized(
+            self.MODEL_ID,
+            device="cuda:0",
+            backend=BACKEND.VLLM,
+            trust_remote_code=True,
+            gpu_memory_utilization=0.2
+        )
+        outputs = model.generate(
+            load_format="vllm",
```

Comment:
Remove `load_format`.  Check if `model.config` is instance of vllm llm_engine.config since it was set in ```model.config = model.llm_engine.model_config```